packages/mobile/src/home/saga.ts

```diff
@@ -42,6 +43,7 @@ export function withLoading<Fn extends (...args: any[]) => any>(fn: Fn, ...args:
 export function* refreshBalances() {
   Logger.debug(TAG, 'Fetching all balances')
   yield call(getConnectedAccount)
+  yield put(fetchTokenBalances())
   yield put(fetchStableBalances())
   yield put(fetchGoldBalance())
   yield put(fetchSentEscrowPayments())
```

Comment:
Hopefully one day we can have a single action 😅

Reply:
😅